### DISCUSSION TITLE
Filtering

### DISCUSSION THREAD

**Discussion by minkkinen** [mgmeyers/obsidian-kanban#47] [Ideas & Feature Requests]
Many thanks for this great plugin. Being able to filter which cards are visible with a text string would make it even more powerful. I could imagine e.g. filtering with a tag or a person’s name to see all cards related to them. This would diminish the need to have separate boards for different projects and make it easier to navigate a growing board.

**Comment by GrupaRX7** [comment]
This is one of the core functionalities that can significantly increase work efficiency. Especially in large projects.
